src/main/java/homework/task_13_queue/coffee/order/CoffeeOrderBoard.java

```diff
@@ -0,0 +1,37 @@
+package homework.task_13_queue.coffee.order;
+
+import java.util.*;
+
+public class CoffeeOrderBoard {
+    private final Queue<Order> queueOrderList = new LinkedList<>();
+    private int nextOrderNumber = 1;
+
+    //1
+    public void add(Order order) {
+        order.setOrderNum(nextOrderNumber);
+        queueOrderList.add(order);
+        nextOrderNumber++;
+    }
+
+    //2
+    public void deliver() {
```

Comment:
You should return needed Order here

Reply:
Updated solved task-13-queue (changed "deliver" methods)